ci/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# "Build Root" definition for providing almost all dependencies
```

Comment:
Should we name this file e.g. `Dockerfile.buildroot`?

OK to do as a follow up.

Reply:
Done :) Easy to rename.

Reply:
Er, nevermind the OpenShift PR merged.